src/test/java/ru/svgogin/service/spark/web/SparkControllerItTest.java

```diff
@@ -249,8 +249,169 @@ void saveCompanyShouldThrowExceptionWhenExists() throws Exception {
         .andExpect(MockMvcResultMatchers.content().json(
                 "{\n"
                 + "    \"code\": \"ERROR001\",\n"
-                + "    \"message\": \"Company with inn 07725038124 already exists\"\n"
+                + "    \"message\": \"Company with inn 7725038124 already exists\"\n"
                 + "}\n"
         ));
   }
+
+  @Test
+  void getCompanyByInnShouldThrowException_WhenPathInnIsNotValid() throws Exception {
```

Comment:
why do you test it in integration test?

Reply:
Tests have been moved to SparkControllerTest